scripts/mkreleaselog

```diff
@@ -1,31 +1,11 @@
 #!/bin/zsh
-set -x
 set -euo pipefail
 export GO111MODULE=on
 export GOPATH="$(go env GOPATH)"
 
 alias jq="jq --unbuffered"
 
-AUTHORS=(
-    # orgs
-    ipfs
-    ipld
-    libp2p
-    multiformats
-    filecoin-project
-    ipfs-shipyard
-
-    # Authors of personal repos used by go-ipfs that should be mentioned in the
-    # release notes.
-    whyrusleeping
-    Kubuxu
-    jbenet
-    Stebalien
-    marten-seemann
-    hsanjuan
-    lucas-clemente
-    warpfork
-)
+AUTHORS=(filecoin-project)
```

Comment:
@Stebalien @magik6k I think the above is a mistaken revert of https://github.com/filecoin-project/lotus/pull/5826
I.e. in `master` we want the full list, no?
cc @BigLep @arajasek

Reply:
Nope, this is a new commit -> https://github.com/filecoin-project/lotus/pull/6583/commits/1411cff69e3dd05dc2be073e104606ab88d34490

Reply:
Right, I believe *that* commit is a mistake